.changeset/brown-donuts-brake.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Go ahead and remove the changeset. We don't track changes to our documentation, only `/package` projects.

Reply:
Gotcha, removed.